css/docs.css

```diff
@@ -5,27 +5,29 @@ flex-direction: column;
 }
 
 .container-docs .navbar{
-    box-shadow: 0px 0px 5px 0px var(--color-pri);
+    box-shadow: 0px 0px 10px 0px var(--bg-color-sec);
 }
 
 
 .main-docs{
+    width: 100%;
     align-items: flex-start;
     color: var(--text-sec);
     position: relative;
+    justify-content: space-between;
 }
 
 /** Sidebar style here */
 
 .sidebar{
-    box-shadow: 0px 5px 10px 0px var(--color-pri);
+    box-shadow: 0px 5px 10px 0px var(--bg-color-sec);
```

Comment:
Box shadow can also be done box-shadow: 5px 10px #888888; for right and down side shadows

Reply:
Refactored that, but I think it's imp to use x-axis and y-axis values in my situation.